sdk/loadtesting/load-testing-rest/src/models.ts

```diff
@@ -1,208 +1,284 @@
+   * input data evenly across all engine instances. If you provide multiple CSV
+   * files, each file will be split evenly.
+   */
   splitAllCSVs?: boolean;
-  /** If true, optionalLoadTestConfig is required and JMX script for the load test is not required to upload. */
+  /**
+   * If true, optionalLoadTestConfig is required and JMX script for the load test is
+   * not required to upload.
+   */
   quickStartTest?: boolean;
-  /** Optional load test config */
+  /** Configuration for quick load test */
   optionalLoadTestConfig?: OptionalLoadTestConfig;
+  /** Region distribution configuration for the load test. */
+  regionalLoadTestConfig?: Array<RegionalConfiguration>;
 }
 
-/** Optional load test config */
+/** Configuration for quick load test */
 export interface OptionalLoadTestConfig {
-  /** Test URL. Provide the complete HTTP URL. For example, http://contoso-app.azurewebsites.net/login */
+  /** Test URL. Provide the complete HTTP URL. For example, https://contoso-app.azurewebsites.net/login */
   endpointUrl?: string;
-  /** No of concurrent virtual users */
+  /** Target throughput (requests per second). This may not be necessarily achieved. The actual throughput will be lower if the application is not capable of handling it. */
+  requestsPerSecond?: number;
+  /** Maximum response time in milliseconds of the API/endpoint. */
+  maxResponseTimeInMs?: number;
+  /** No of concurrent virtual users. */
   virtualUsers?: number;
-  /** Ramp up time */
+  /** Ramp up time in seconds. */
   rampUpTime?: number;
-  /** Test run duration */
+  /** Test run duration in seconds. */
   duration?: number;
 }
 
+/** Region distribution configuration for the load test. */
+export interface RegionalConfiguration {
+  /**   The number of engine instances to execute load test in specified region. Supported values are in range of 1-400. */
+  engineInstances: number;
+  /**
+   * Azure region name.
+   * The region name should of format accepted by ARM, and should be a region supported by Azure Load Testing. For example, East US should be passed as "eastus".
+   * The region name must match one of the strings in the "Name" column returned from running the "az account list-locations -o table" Azure CLI command.
+   */
+  region: string;
+}
+
 /** The input artifacts for the test. */
 export interface TestInputArtifacts {
   /** File info */
-  configFileInfo?: FileInfo;
+  configFileInfo?: TestFileInfo;
   /** File info */
-  testScriptFileInfo?: FileInfo;
+  testScriptFileInfo?: TestFileInfo;
   /** File info */
-  userPropFileInfo?: FileInfo;
+  userPropFileInfo?: TestFileInfo;
   /** File info */
-  inputArtifactsZipFileInfo?: FileInfo;
-  /** Additional supported files for the test run */
-  additionalFileInfo?: Array<FileInfo>;
+  inputArtifactsZipFileInfo?: TestFileInfo;
+  /** The config json file for url based test */
+  urlTestConfigFileInfo?: TestFileInfo;
 }
 
-/** File info */
-export interface FileInfo {
```

Comment:
this model is missing? should it be a breaking change? which means we probably should bump major version.

Reply:
It is not missing, it is renamed to `TestFileInfo` 

![image](https://github.com/user-attachments/assets/bd11e1a4-3f81-43fe-a1a4-0ca0fa6f47fb)

There hasnt been any change in this area